plugins/navtiming.js

```diff
@@ -364,7 +364,7 @@
 				// Remove any properties that are undefined
 				for (k in data) {
 					if (data.hasOwnProperty(k) && data[k] === undefined) {
-						delete data[k];
+						data[k] = null;
```

Comment:
This doesn't make sense. We're changing the value from `undefined` to `null`.

Reply:
Same here - that delete is strict-friendly. Not sure what I was thinking there.